server/testing/src/main/java/org/apache/james/util/docker/Images.java

```diff
@@ -23,7 +23,7 @@
     String FAKE_SMTP = "weave/rest-smtp-sink:latest";
     String RABBITMQ = "rabbitmq:3.7.7-management";
     String ELASTICSEARCH_2 = "elasticsearch:2.4.6";
-    String ELASTICSEARCH_6 = "elasticsearch:6.7.2";
+    String ELASTICSEARCH_6 = "quay.io/pires/docker-elasticsearch-kubernetes:6.3.2";
```

Comment:
On my local even the start of that docker image was crashing... I would suspect it being more adapted to k8s usage? (although I didn't check). Probably better to use the image pointed out by @rouazana : docker.elastic.co/elasticsearch/elasticsearch:6.3.2

https://www.docker.elastic.co